packages/dev/sharedUiComponents/src/fluent/primitives/dropdown.tsx

```diff
@@ -9,18 +9,36 @@ const useDropdownStyles = makeStyles({
     optionsLine: {},
 });
 
-export const Dropdown = (props: { options: IInspectableOptions[]; onSelect: (o: string) => void; defaultValue: string | number }) => {
+export type DropdownOption = {
+    /**
+     * Defines the visible part of the option
+     */
+    label: string;
+    /**
+     * Defines the value part of the option (returned through the callback)
+     */
+    value: string | number;
+};
+
+type DropdownProps = { options: readonly DropdownOption[]; onSelect: (o: string) => void; defaultValue?: DropdownOption };
+
+/**
+ * Renders a fluent UI dropdown with a calback for selection and a required default value
+ * @param props
+ * @returns dropdown component
+ */
+export const Dropdown: FunctionComponent<DropdownProps> = (props) => {
     const styles = useDropdownStyles();

```

Comment:
Right now there is a mix of `const styles =` and `const classes =`. I would use the latter, since that is how the Fluent docs show it, so it is easier for people to immediately recognize it.